rhg_compute_tools/gcs.py

```diff
@@ -316,3 +316,20 @@ def sync(src, dest, flags=["r", "d"]):
     end_time = dt.now()
 
     return stdout, stderr, end_time - st_time
+
+
+def ls(dir_path):
+    """List a directory quickly using `gsutil`"""
+
+    dir_url = str(dir_path).replace("/gcs/", "gs://")
+
+    cmd = f"gsutil ls {dir_url}"
+
+    print(f"Running cmd: {cmd}")
+    cmd = shlex.split(cmd)
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = p.communicate()
+
+    res = [x.split("/")[1].rstrip(r"\\n") for x in str(stdout).split(dir_url)[2:]]
+
+    return res
```

Comment:
Minor thing. A linter might complain about needing a blank/newline at the end of the file...? 

(following up after looking at gh-actions config.) Looks like the linter has beef with a couple files in the repo but it's not failing the CI job, so I'm not sure the priority on lint-stuff.

Reply:
done! Newline added but I'll leave the other files alone for now. We could do one isolated PR where we lint/format stuff if we want to clean that up at some point

Reply:
@bolliger32, yeah I think that's a good idea. It's def a separate PR.

Is CI/CD supposed to flag commits with failed black or flake8 checks? Right now it's passing even though the linters are recommending changes. I don't know if this is intentional or not.

Reply:
I don't think it was originally set up that way but that might be a @delgadom question. I wouldn't be opposed to having it raise an error with a black or flake8 check (probably black would be my thought as this would be a very easy fix), but not sure if we want to impose that strictness